website_sale_suggest_create_account/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================================
+Suggest to create user account when buying
+==========================================
+
+This module was written to extend the functionality of e-Commerce to suggest
+the buyer to create an account, but without forcing him to do it.
+
+This way you can avoid duplication of partners in your database and offer your
+users a better experience without blocking 1-time buyers.
+
+However, the *Checkout without sign in* button is not highlighted, to
+implicitly encourage users to create the account.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* *Settings > Configuration > General Settings* and enable *Allow external
```

Comment:
Allowing anybody to sign into your Odoo implies some security considerations, so I think this step should be made by hand. Anyway this parameter has no XMLID to inherit from, so does not seem possible to do it with data files.